include/smack/SplitStructLoadStore.h

```diff
@@ -0,0 +1,31 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+
+#include "llvm/IR/DataLayout.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Pass.h"
+#include "llvm/IR/IRBuilder.h"
+
+#include <vector>
+#include <utility>
+
+namespace smack {
+
+class SplitStructLoadStore : public llvm::ModulePass {
+public:
+  static char ID;
+
+  SplitStructLoadStore() : llvm::ModulePass(ID) {}
+  virtual bool runOnModule(llvm::Module& M);
+private:
+  const llvm::DataLayout* DL;
+  std::vector<llvm::Instruction*> toRemove;
```

Comment:
Couldn't this be a local variable — e.g., in what is currently the `runInModule` function — rather than an instance variable?